mypy.ini

```diff
@@ -12,6 +12,8 @@ warn_unused_ignores = True
 warn_redundant_casts = True
 warn_unused_configs = True
 warn_unreachable = True
+disallow_untyped_defs = true
+disallow_incomplete_defs = true
```

Comment:
This makes lots of tests fail because test code in `*.yml` is missing annotations

```
-   case: transaction_atomic_decorator
    main: |
            ...
            @transaction.atomic(savepoint=True)
            def atomic_method2(self):
```

I can add missing hints to test functions too.

Reply:
How many functions there are? If you have time for this - it is very useful!
If you don't - we can for now use different settings for tests / code.

Reply:
There aren't too many, but I'll submit a separate PR for clarity.